src/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: DocumentationLayout
+pageHeading: Getting Started
+---
+
+# Getting Started with running SRI - Stratum v2 reference implementation
+
+SRI stack is flexible. It allows you to run a few different configurations. The easiest way to test these configurations is to use the SRI [role](https://github.com/stratum-mining/stratum/tree/main/roles) implementations.
+
+Below are the most commonly used configurations you can run to get started.
+
+## Config 1: SV1 firmware > Translation Proxy > SV2 Pool
+
+This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). The proxy is designed to sit in between a SV1 downstream role (most typically Mining Device(s) running SV1 firmware) and a SV2 upstream role (most typically a SV2 Pool Server).
+
+![Config1](/assets/config-c.svg)
+
+### 1. Start Pool
+
+The Pool role should be configured to point to the hosted Template Provider. In the `pool-config.toml` file you should see this: `tp_address = "75.119.150.111:8442"` The default `pool-config.toml` should have appropriate defaults set up for everything else.
+
+1. `cd roles/v2/pool`
+2. Then `cargo run -p pool`
+
+If the pool properly starts you should see the following log lines:
+
+```
```

Comment:
can add code highlighting here
```suggestion
``` log
```

Reply:
This can be done for all code snippets
<img width="619" alt="Screen Shot 2023-02-22 at 12 13 40 PM" src="https://user-images.githubusercontent.com/14062229/220719118-36fbfb42-1a8c-4244-b1e4-9198c7e80322.png">

Reply:
Thanks for pointing that out @joshr4 I missed that, but still it doesn't seem to highlight this for example very clearly, any ideas on how to improve that?

![Screenshot 2023-02-22 at 19 45 24](https://user-images.githubusercontent.com/36959754/220728493-30be270f-a916-442c-a0c1-e8aad7c33323.png)

Reply:
Oh good point, I'm not sure how to get the that work for the inline code snippets. I'm new to Vue and need to research more on how this is all produced from the markdown file.

If we could directly edit the html that is produced, we could manually add in the class that makes this render as code. I manually added `class="language-shell"` to test this out:
<img width="759" alt="Screen Shot 2023-02-22 at 2 44 52 PM" src="https://user-images.githubusercontent.com/14062229/220755015-2ffb15a1-0198-4b73-bfe9-c7205bedcbe6.png">